lib/ts/station/station.ex

```diff
@@ -6,14 +6,17 @@ defmodule Station do
   @behaviour Station.StationBehaviour
   use GenServer
   require Station.FSM
+  require Logger
   alias Station.FSM, as: FSM
 
   # Client-Side functions
 
   # Starting the GenServer
 
   def start_link(station_data) when is_list(station_data) do
-    GenServer.start_link(Station, station_data)
+    pid_tuple = GenServer.start_link(Station, station_data)
+    Logger.info(fn -> "Station started at pid=#{inspect elem(pid_tuple,1)}" end)
```

Comment:
The pid is not known reliably before the completion of execution for init/1. Hence, the above logger statement might not always give reliable result.